api/conf/asgi.py

```diff
@@ -15,8 +17,21 @@ def get_asgi_application():
     return OpenverseASGIHandler()
 
 
-application = get_asgi_application()
-
+# Give the base application a unique name so that it's easily referenced
+# by other modules without needing to juggle through any of the various
+# middlewares.
+# For example, `ASGIStaticFileHandler` saves the application as
+# `self.application`. However, `SentryAsgiMiddleware` saves it as `self.app`.
+# If we relied on just referencing the pseudo-standard `application` exported
+# in this module, we'd have to choose how to resolve the base application
+# with the lifecycle handlers depending on what middleware was applied.
+# Instead, we can just take advantage of the fact that these middleware
+# all treat the passed-in application object as a singleton, so we're safe
+# to directly reference `OPENVERSE_APPLICATION`, knowing that any middleware
+# applied to it will be referencing the same application object in the end.
+OPENVERSE_APPLICATION = get_asgi_application()
```

Comment:
This is in anticipation of #3024, which is the primary reason we need the lifecycle callbacks.